test/hotspot/jtreg/compiler/c2/TestCastX2NotProcessedIGVN.java

```diff
@@ -62,7 +62,8 @@ public static void test1Runner() {
     }
 
     @Test
-    @IR(counts = {IRNode.LOAD_VECTOR_I, "> 1"})
+    @IR(counts = {IRNode.LOAD_VECTOR_I, "> 1"},
+        applyIfPlatformOr = {"x64", "true", "aarch64", "true", "riscv64", "true"})
```

Comment:
Hi, Could you please remove `riscv64` from this line? I just found that this test also fails when testing on riscv64 platforms where the vector extension is not available. Thanks.

Reply:
Sure. I've removed it.